docs/tutorials/pre_executed/ztf-alerts-sne.ipynb

```diff
@@ -0,0 +1,2117 @@
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>oid</th>\n",
+       "      <th>mean_ra</th>\n",
+       "      <th>mean_dec</th>\n",
+       "      <th>Norder</th>\n",
+       "      <th>Dir</th>\n",
+       "      <th>Npix</th>\n",
+       "      <th>lc</th>\n",
+       "      <th>nondet</th>\n",
+       "      <th>ref</th>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>npartitions=113</th>\n",
+       "      <th></th>\n",
+       "      <th></th>\n",
+       "      <th></th>\n",
+       "      <th></th>\n",
+       "      <th></th>\n",
+       "      <th></th>\n",
+       "      <th></th>\n",
+       "      <th></th>\n",
+       "      <th></th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>string[pyarrow]</td>\n",
+       "      <td>double[pyarrow]</td>\n",
+       "      <td>double[pyarrow]</td>\n",
+       "      <td>int8[pyarrow]</td>\n",
+       "      <td>int64[pyarrow]</td>\n",
+       "      <td>int64[pyarrow]</td>\n",
+       "      <td>nested&lt;lc_ra: [list&lt;element: double&gt;], lc_dec:...</td>\n",
+       "      <td>nested&lt;nondet_mjd: [list&lt;element: double&gt;], no...</td>\n",
+       "      <td>nested&lt;ref_rfid: [list&lt;element: int64&gt;], ref_c...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>72057594037927936</th>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>...</th>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3170534137668829184</th>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3458764513820540928</th>\n",
```

Comment:
It is super confusing that these two catalogs have the same name

Reply:
Yes, but it is out of scope...